admin/meetings/urls.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import absolute_import
+
+from django.conf.urls import url
+
+from admin.meetings import views
+
+urlpatterns = [
+    url(r'^$', views.MeetingListView.as_view(), name='list'),
+    url(r'^create/$', views.MeetingCreateFormView.as_view(), name='create'),
```

Comment:
Prefer RESTful routing. You shouldn't need a `/meetings/create/` endpoint; just `POST` to `/meetings/`.